src/tile.cpp

```diff
@@ -1612,15 +1600,15 @@ Item* Tile::getUseItem(int32_t index) const
 	}
 
 	// try getting thing by index
-	if (Thing* thing = getThing(index)) {
-		Item* thingItem = thing->getItem();
+	if (auto thing = getThing(index)) {
```

Comment:
Could this be `const auto`
```suggestion
	if (const auto thing = getThing(index)) {
```